src/js/components/Select/StyledSelect.js

```diff
@@ -1,18 +1,74 @@
 import styled from 'styled-components';
 
-import { sizeStyle } from '../../utils';
+import { Button } from '../Button';
+import { DropButton } from '../DropButton';
+import { TextInput } from '../TextInput';
+import {
+  getHoverIndicatorStyle,
+  selectedStyle,
+  controlBorderStyle,
+  sizeStyle,
+} from '../../utils';
 
 export const StyledContainer = styled.div`
   /* IE11 hack to get drop contents to not overflow */
   @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
     width: 100%;
   }
 
-  ${props =>
+  ${(props) =>
     props.dropHeight
       ? sizeStyle('max-height', props.dropHeight, props.theme)
       : 'max-height: inherit;'};
 
-  ${props =>
+  ${(props) =>
     props.theme.select.container && props.theme.select.container.extend};
 `;
+
+// position relative is so scroll can be managed correctly
+export const OptionsBox = styled.div`
```

Comment:
Since this isn't actually a "Box", can we call it something like "OptionsContainer"?

Reply:
Yes, done